src/texdocdialog.cpp

```diff
@@ -140,7 +143,28 @@ void TexdocDialog::itemChanged(QTableWidgetItem* item)
 	openButton->setEnabled(false);
 	int row = item->row();
 	QString text = ui->tbPackages->item(row,0)->text();
-	delayedCheckDocAvailable(text);
+    LatexPackageInfo package = item->data(Qt::UserRole).value<LatexPackageInfo>();
+    QString Info = LatexRepository::packageInfo(package);
+    ui->packageInfoBrowser->setMarkdown(Info);
+    delayedCheckDocAvailable(text);
+    buttonGroup.setExclusive(true);
+
+    while(buttonGroup.buttons().count()>0){
+        QAbstractButton * bt = buttonGroup.buttons().first();
+        ui->LanguagesLayout->removeWidget(bt);

```

Comment:
LanguageLayout should start with lower case

Reply:
Missed that. I will fix it right away